packages/js/src/types.ts

```diff
@@ -99,6 +118,29 @@ export type Cta = {
   action?: MessageAction;
 };
 
+export type WorkflowInfo = {
+  _id: string;
+  name: string;
+  critical: boolean;
+  tags?: string[];
+  identifier: string;
```

Comment:
What's the difference between an `identifier` and the `_id`?

Reply:
`identifier` is used for the trigger, `_id` is internal Mongo id